NOTES.md

```diff
@@ -1,3 +1,20 @@
+# Release 2.0.3 (Yojimbo)
+
+Released on 2015-06-10
+
+Bug fix release
+
+* Fixed a bug that broke autocompletion in the Data Explorer (№4261)
+* No longer crash when overflowing the stack in a query (#2639)
```

Comment:
Since this is still not handling all known cases, I'd prefer:
"No longer crash for certain types of stack overflows during query execution (#2639)"